packages/drivers/local-driver/src/localResolver.ts

```diff
@@ -66,9 +66,7 @@ export class LocalResolver implements IUrlResolver {
 		if (url.startsWith("/")) {
 			url = url.substr(1);
 		}
-		const fluidResolvedUrl = resolvedUrl as IFluidResolvedUrl;
-
-		const parsedUrl = parse(fluidResolvedUrl.url);
+		const parsedUrl = parse(resolvedUrl.url);
```

Comment:
Love these little improvements!  Way easier to read for someone new to the code.